# No Title

[Original Post](https://discourse.onlinedegree.iitm.ac.in/t/164277/216)

<p><a class="mention" href="/u/jivraj">@Jivraj</a> sir I get this error<br>
but my app.py is able to get the server running on localhost and not on 0.0.0.<br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/e/d/ed519f25f712a007f48e1e2f3cf5cf7f946271cb.png" data-download-href="/uploads/short-url/xRq27aO3iKC8e2tH9JXnzpGWF0T.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/e/d/ed519f25f712a007f48e1e2f3cf5cf7f946271cb.png" alt="image" data-base62-sha1="xRq27aO3iKC8e2tH9JXnzpGWF0T" width="690" height="129" data-dominant-color="222425"><div class="meta"><svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1014Ã—190 18.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg></div></a></div><br>
could you please help ?</p>

![Image](https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/e/d/ed519f25f712a007f48e1e2f3cf5cf7f946271cb.png)